ballerina/http_response.bal

```diff
@@ -521,7 +522,7 @@ public class Response {
         } else if payload is stream<SseEvent, error?> {
             self.setSseEventStream(payload);
         } else if payload is anydata {
-            self.setJsonPayload(payload.toJson());
+            self.setJsonPayload(jsondata:toJson(payload));
```

Comment:
Same as the request method. Have a separate function for setting anydata payload as json payload